src/ui/internal-page/InternalPage.js

```diff
@@ -60,6 +62,39 @@ const renderDigitalExchangeMenuItem = () => {
   return '';
 };
 
+const renderCMSMenuItems = () => {
+  if (process.env.CMS_UI_ENABLED) {
```

Comment:
please use object destructuring to get flags from `process.env` (e.g. you can use `CMS_UI_ENABLED` instead of `process.env.CMS_UI_ENABLED`)

please use ternary operator instead of explicit `if`